fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/roles/ldap/LdapUserRolesProvider.java

```diff
@@ -94,7 +94,9 @@
             params,
             configProps.getGroupRoleAttributes());
 
-    log.debug("Got roles for user " + userId + ": " + userRoles);
+    if (log.isDebugEnabled()) {
+      log.debug("Got roles for user " + userId + ": " + userRoles);
```

Comment:
I'd think the if check here for debug is redundant, as log levels would control whether this is output or not?

Reply:
If you're worried about the performance of the log statement, you'd use something like

```java
       log.debug("Got roles for user {}: {}", userId, userRoles); 
```

Which will at least skip the string construction.

Reply:
fixed